data/scripts/berry_tree.inc

```diff
@@ -2,142 +2,142 @@ BerryTreeScript:: @ 82742F9
 	special EventObjectInteractionGetBerryTreeData
 	switch VAR_0x8004
 	case 255, Route102_EventScript_27434F
```

Comment:
Since they're shared between all berry trees, they shouldn't use the Route102 prefix anymore.